editor.planx.uk/src/@planx/components/Checklist/Public/tests/mockOptions.ts

```diff
@@ -63,6 +63,21 @@ export const options: {
     },
   ],
 };
+
+const newMockOptions = [];
+if (options[ChecklistLayout.Basic]) {
+  newMockOptions.push(...options[ChecklistLayout.Basic]);
+  newMockOptions.push({
+    id: "tent_id",
+    data: {
+      text: "Tent",
+      exclusive: true,
+    },
+  });
+}
+
+export const optionsWithExclusiveOption: Array<Option> = newMockOptions;
```

Comment:
This is a bit unwieldy, but when I tried to do

```js
export const optionsWithExclusiveOption: Array<Option> = [...options[ChecklistLayout.Basic], {
    id: "tent_id",
    data: {
      text: "Tent",
      exclusive: true,
    },
  }];

```

I got this type error: `Type 'Option[] | undefined' must have a '[Symbol.iterator]()' method that returns an iterator.ts`